scripts/run_python_test.sh

```diff
@@ -44,15 +44,16 @@ if ! type "parallel" 2> /dev/null; then
         if [[ $OSTYPE != msys ]]; then
           cmd=${PYTHON_EXECUTABLE}
         fi
-        cmd="${cmd} ${PYTEST_EXECUTABLE} -v $PYTHONPATH/turicreate/test --junit-xml=alltests.pytest.xml"
-        echo $cmd
+        cmd="${cmd} ${PYTEST_EXECUTABLE} --cov -v $PYTHONPATH/turicreate/test \
+          --junit-xml=alltests.pytest.xml"
+        echo "$cmd"
```

Comment:
You probably want to remove this `echo`. Especially since `set -x` is included at the top of the script.

Reply:
That's right! will do.